### PR TITLE
fixing `Wunused-parameter` warnings

### PR DESCRIPTION
**Pull request by vissarion**

### CODE REVIEW

File: include/boost/geometry/strategies/cartesian/buffer_end_round.hpp [hunk 1]
```diff
@@ -116,6 +116,7 @@ public :
                 DistanceStrategy const& distance,
                 RangeOut& range_out) const
     {
+        boost::ignore_unused(perp_left_point);
```
Comment (mloskot) [inline include/boost/geometry/strategies/cartesian/buffer_end_round.hpp]: Shouldn't there be explicit `#include <boost/core/ignore_unused.hpp>` for this, to keep the header self-contained, etc.?

Comment (vissarion) [reply include/boost/geometry/strategies/cartesian/buffer_end_round.hpp]: Yes, this is better. Changed. Thanks!

**Comment by awulkiew** [comment]
Thanks!

File: include/boost/geometry/strategies/cartesian/buffer_end_round.hpp (hunk 1 above)
Comment (barendgehrels) [reply include/boost/geometry/strategies/cartesian/buffer_end_round.hpp]: Thanks!

**Pull request merged**